codespell_lib/data/dictionary.txt

```diff
@@ -9065,9 +9065,18 @@ imigrated->emigrated, immigrated,
 imigration->emigration, immigration,
```

Comment:
Likewise here, I assume they've got their e/i right and more likely to have missed a letter.

Reply:
0ce5399